conbench/templates/compare-list.html

```diff
@@ -125,80 +125,66 @@ <h6>Top Outliers</h6>
                   <div>{{ c.baseline.benchmark_name }}</div>
                 </td>
                 <td>
-                  {% if (c.contender.value is not none or c.contender_error) and (c.baseline.value is not none or c.baseline_error) %}
-                    <a href="{{ c.compare_benchmarks_url }}">
-                      <div>{{ c.baseline.case_permutation }}</div>
-                    </a>
-                  {% else %}
-                    <div></div>
-                  {% endif %}
+                  <a href="{{ c.compare_benchmarks_url }}">
+                    <div>{{ c.baseline.case_permutation }}</div>
+                  </a>
                 </td>
                 {% if c.baseline.error is not none %}
                   <td>
-                    <a href="{{ url_for('app.benchmark-result', benchmark_result_id=c.baseline.benchmark_result_id) }}">
-                      <i class="glyphicon glyphicon-exclamation-sign text-danger"
-                         data-toggle="tooltip"
-                         data-placement="top"
-                         title="Has error">
-                      </i></a>
-                    </td>
-                  {% else %}
-                    <td style="white-space: nowrap;">{{ c.baseline.value }} {{ c.unit }}</td>
-                  {% endif %}
-                  {% if c.contender.error is not none %}
-                    <td>
-                      <a href="{{ url_for('app.benchmark-result', benchmark_result_id=c.contender.benchmark_result_id) }}">
-                        <i class="glyphicon glyphicon-exclamation-sign text-danger"
-                           data-toggle="tooltip"
-                           data-placement="top"
-                           title="Has error">
-                        </i></a>
-                      </td>
-                    {% else %}
-                      <td style="white-space: nowrap;">{{ c.contender.value }} {{ c.unit }}</td>
-                    {% endif %}
-                  </tr>
+                    <a href="{{ url_for('app.benchmark-result', benchmark_result_id=c.baseline.benchmark_result_id) }}">Error</a>
```

Comment:
The glyph doesn't seem to be working; not sure if it's related to recent dependency upgrades. I chose to just display the word "Error" instead.

Reply:
💯

Reply:
Thanks for the explanation! :)